pkg/workflows/wasm/host/module.go

```diff
@@ -401,31 +429,42 @@ func containsCode(err error, code int) bool {
 	return strings.Contains(err.Error(), fmt.Sprintf("exit status %d", code))
 }
 
-func fetchFn(logger logger.Logger, modCfg *ModuleConfig) func(caller *wasmtime.Caller, respptr int32, resplenptr int32, reqptr int32, reqptrlen int32) int32 {
-	const fetchErrSfx = "error calling fetch"
+func fetchFn(logger logger.Logger, modCfg *ModuleConfig, store *store) func(caller *wasmtime.Caller, respptr int32, resplenptr int32, reqptr int32, reqptrlen int32) int32 {
+	logFetchErr := func(err error) { logger.Errorf("error calling fetch: %s", err.Error()) }
 	return func(caller *wasmtime.Caller, respptr int32, resplenptr int32, reqptr int32, reqptrlen int32) int32 {
 		b, innerErr := wasmRead(caller, reqptr, reqptrlen)
 		if innerErr != nil {
-			logger.Errorf("%s: %s", fetchErrSfx, innerErr)
+			logFetchErr(innerErr)
```

Comment:
🤔 you might be right. I had some conflicts when merging main branch and might have brought back this by mistake. checking...

Reply:
info for reviewers: this hasn't been addressed before, it will be done on this new task [CAPPL-195](https://smartcontract-it.atlassian.net/browse/CAPPL-195)